about.html

```diff
@@ -0,0 +1,174 @@
+                <a href="index.html" class="nav-link">Home</a>
+            </li>
+            <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+            </li>
+            <li class="nav-item">
+                <a href="" class="nav-link">Program</a>
+            </li>
+            <li class="nav-item">
+                <a href="" class="nav-link">Join</a>
+            </li>
+               <li class="nav-item labelT">
+              <a href="" class="nav-link">WFH Remoters..</a>
+          </li><img src="images/crane.jfif" alt="" class="crane1"> <img src="images/crane1.jfif" alt="" class="crane2">
+          </ul>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+        </nav>
+        </header>
+        <!--End of Header section-->
+    </header>
+    <!--Header section-->
+    <section class="mainS"></section>
+           <div class="main-header">
+            <p class="hello">"Hello! Am a WFH"</p>
+                   <h1 class="header-text">
+                        REMOTE DEV'S ANNUAL GLOBAL &nbsp;SUMMIT 2022 
+          </h1>
+          <p class="desc"> The after month of covid 19 saw an over whelming growth of the demand for remote workers
+            Previously a very slow sector experienced a sharp demand as companies had also come to appreciate thie benefits associtaed. the initially 
+            unknown event now gathers all works of life with a turn up of thousands each year. For the first Time, its in AFRICA , Kampala, Uganda in E.Africa this OCTOBER</p>
+             <p class="desc2">Please contact @www.MunyonyoCommonwealth esort.ug, Along Lake Victoria Shores, @www.TripeAdvisors.com, www.Eaafricansafari.com</p>
+             </div>
+        <div class="middle">
+             <div class="mid-label">
+              <!-- <div class="footer-head ">
+                <a href="#"> 
+                  <img class="logo-img" src="images/proLogo.png" alt="logo"/></a>
+                   <p>
+                Ⓒ 2022 WFH Uganda <br />
+                All Rights Reserved
+              </p>
+              </div> -->
+            
+                  <!-- <h3>REMOTE <br>GLOBAL <br>SUMMIT</h3> -->
+                  
+             <h3 class="pearl">@ Pearl</h3>
+             <img src="images/Equator.jpg" alt="The Equater" class="pearl-img" title="@ the Equater">
+             <img src="images/Pearl-of-Africa-Hotel.jpg" alt="Speke Resort Hotel" class="pearl-img" title="Speke Resort Hotel">
+            
+          </div>
+          </div>
+    <!--End of Header section-->
+
+    <!--Middle section-->
+    <div class="middle">
+              <h2 class="date">2022.10.15(MON ~ 17(WED)</h2>
+              <h3>REMOTE DEV's GLOBAL SUMMIT ~2022</h3>
+              <p>
+                Kampala, Uganda (The Pearl of AFRICA) <br />
+                Its has never been better
+              </p>
```

Comment:
- [ ] Kindly edit this section to make it match the one in the guidelines. Please add a picture of the logo and the text that matches the context of the text in the guidelines 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190241217-092ad08b-68d4-41c5-a8e4-fd02db498705.png)

## Guidlines:
![image](https://user-images.githubusercontent.com/50721479/190241258-41f858f5-fce7-47a9-8788-47acab669025.png)